server/models/availability.js

```diff
@@ -0,0 +1,54 @@
+const mongoose = require("mongoose");
+
+
+const dayInformation={
+    isAvaliable:{
+        type:Boolean,
+        default:'true'
+    },
+    startDate:{
+        type:Number,
+        min:0,
+        max:24,
+        required:isAvaliableForDay
+    },
+    endDate:{
+        type:Number,
+        min:0,
+        max:24,
+        required:isAvaliableForDay,
+        validate: [dateValidator, 'Start Date must be less than End Date']
+    }
+}
+function isAvaliableForDay(){
+    if(this.isAvaliable > -1){  
+        return true;
+    }
+    return false;
+}
+
+const availabilitySchema = new mongoose.Schema({
+  petSitterId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: 'Profile'
+  },
+  name:{
+      type:'string',
+      required:'true'
+  },
+  days: {
+  sunday:dayInformation,
+  monday:dayInformation,
+  tuesday:dayInformation,
+  wednesday:dayInformation,
+  thursday:dayInformation,
+  friday:dayInformation,
+  saturday:dayInformation
+  },
```

Comment:
The indentation is off here, I highly recommend setting up [`Prettier`](https://prettier.io/) with your favorite editor/ide👍🏼

Reply:
@panwaramita The indentation still seems off, maybe configure your code editor to format the document on save?

If you are using VSCode: use Command + , on Mac or Control + , on Windows to open the settings menu. Then search for Editor: Format on Save and make sure it is checked.

Reply:
Done the formatting.